x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/messages/Messages.java

```diff
@@ -147,6 +147,8 @@
     public static final String JOB_CONFIG_UPDATE_ANALYSIS_LIMITS_MODEL_MEMORY_LIMIT_CANNOT_BE_DECREASED =
             "Invalid update value for analysis_limits: model_memory_limit cannot be decreased below current usage; " +
                     "current usage [{0}], update had [{1}]";
+    public static final String JOB_CONFIG_DUPLICATE_DETECTORS_DISALLOWED =
+            "Specifying duplicate detectors is disallowed, {0}";
```

Comment:
nit: Perhaps `Duplicate detectors are not allowed` would be simpler? It sounds a bit more natural to me.

Reply:
Done. Also, please advise on the location of '{0}' placeholder.